src/detectors/nosql_injection/nosql_injection.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=nosql-injection@v1.0 defects=1}
+var AWS = require('aws-sdk')
+var express = require("express")
+var app = express()
+function noSqlInjectionNoncompliant() {
+    app.get(path, function(req,res) {
+        var dobClient = new AWS.DynamoDB({apiVersion: '2012-08-10'})
+        var params= req.body.params
+        // Noncompliant: external user input can be vulnerable to injection attacks.
+        dobClient.query(params, function(err, data) {
```

Comment:
Can we use `docClient.scan` to keep it consistent?

Reply:
Updated